documentation/release_notes.markdown

```diff
@@ -10,6 +10,18 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.3.0
+
+Released 26 March 2019
+
+### New features
+
+- Puppet Server has a new endpoint for catalog retrieval, allowing more options than the previous endpoint. This endpoint is controlled by `tk-auth`, and by default is not generally accessible. It is intended for use by other Puppet services. For details on the API, see the [Puppet API catalog](https://github.com/puppetlabs/puppetserver/blob/master/documentation/puppet-api/v4/catalog.markdown). [SERVER-2434](https://tickets.puppetlabs.com/browse/SERVER-2434) 
```

Comment:
I think it'd be good to note that this is an API that integrators can hit to provide `puppet master --compile` like functionality, a la https://tickets.puppetlabs.com/browse/PUP-9055

Reply:
I updated it, let me know if that works. Or do you also think we should link to PUP-9055?

Reply:
I was debating that, I think a link might be nice actually.